CAM/SetTableGamesCameraThisUpdate.md

```diff
@@ -0,0 +1,40 @@
+---
+ns: CAM
+aliases: ["_0x79C0E43EB9B944E2"]
+---
+## SET_TABLE_GAMES_CAMERA_THIS_UPDATE
+
+```c
+// 0x79C0E43EB9B944E2
+BOOL SET_TABLE_GAMES_CAMERA_THIS_UPDATE(Hash hash);
+```
+
+Override to table game camerawork for current frame only.
+
+```lua
+eTableGames = {
+  LuckeyWheel = 5891389,
+  Slots = 518572876,
+  Roulette = 71681063,
+  Blackjack = -2124244681,
+  ThreeCard = -1938411241,
+}
+```
```

Comment:
Table method is: https://github.com/citizenfx/natives/commit/12f272480700b1a65b0b4be45caf3d506f4a8aa9#diff-fe04c88c56a62288febcdf5b621e3aed74624d92f1434aecc868e7aed0ee8864R17-R25

So do:

```
|   Hash        |    Game         |
| :-----------:| :-------------:  |
|   `5891389`   |  Lucky Wheel    |
|  `518572876`  |     Slots       |
|   `71681063`  |    Roulette     |
| `-2124244681` |    Blackjack    |
| `-1938411241` |   Three Cards   |
```

Reply:
I see! I didn't understand what you meant by table method, you mean write it in markdown.